pkgs/racket-doc/scribblings/reference/hashes.scrbl

```diff
@@ -212,6 +212,13 @@ mappings can hide earlier mappings.
 
 See also @racket[make-custom-hash].
 
+@examples[
+#:eval the-eval
+(make-hash)
+(make-hash '((0 1) (1 2) (2 3)))
```

Comment:
Do you really intend to have `'(1)`, `'(2)`, and `'(3)` as the values here?

Reply:
No, the reason I made this change was because I couldn't find the assocs written anywhere. Is there a way to make the values just 1, 2, 3?

Reply:
`'([0 . 1] [1 . 2] [2 . 3])` would be a way to do it. Alternatively, you could also do `(list (cons 0 1) (cons 1 2) (cons 2 3))`, which is more general, since the key/values don't need to be `quote`-able.

Reply:
Thanks, I appreciate the examples. I have updated the docs to show various ways to instantiate initial values